.dockerignore

```
@@ -1,4 +1,8 @@
 .git
+!/.git/HEAD
+!/.git/refs/heads/master
+!/.git/refs/heads/test
```

```suggestion
!/.git/refs/heads
```
no harm in having all refs, which will do other branches e.g. on prs